plugins/python-build/bin/python-build

```diff
@@ -925,6 +936,13 @@ fix_jruby_shebangs() {
   done
 }
 
+build_package_truffleruby() {
```

Comment:
Can be removed?

Reply:
At least I'm pretty sure it's some unused code and we can safely remove it. I ended up to import them as well to use it as placeholder when diffing between ruby-build's impl...